### PR TITLE
Adding removeInstances method for a job -

### PR DESCRIPTION
**Pull request by sunilmvn**
-- This method takes the job key and the number of Instances to remove as parameters

### CODE REVIEW

File: examples/client.go [hunk 1]
```diff
@@ -273,15 +273,50 @@ func main() {
...
+
+		resp, err := r.RemoveInstances(job.JobKey(), numOfInstances)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if resp.ResponseCode == aurora.ResponseCode_OK {
+			if ok, err := monitor.Instances(job.JobKey(), currInstances-numOfInstances, 5, 50); !ok || err != nil {
 				fmt.Println("Flexing up failed")
```
Comment (rdelval) [inline examples/client.go]: Should be flex down failed

Comment (kkrishna) [inline examples/client.go]: fixed.. good catch.. Renan!

File: realis.go [hunk 2]
```diff
@@ -320,6 +322,28 @@ func (r realisClient) AddInstances(instKey aurora.InstanceKey, count int32) (*au
 	return response.ResponseCodeCheck(resp)
 }
 
+//Scale down the number of instances under a job configuration using the configuratipn of a specific instance
```
Comment (rdelval) [inline realis.go]: Nit: configuratipn -> configuration

File: realis.go [hunk 3]
```diff
@@ -320,6 +322,28 @@ func (r realisClient) AddInstances(instKey aurora.InstanceKey, count int32) (*au
...
+	if err != nil {
+		return nil, errors.Wrap(err, "RemoveInstances: Could not retrieve relevant instance IDs")
+	}
+	if len(instanceIds) < int(count) {
+		return nil, errors.New(fmt.Sprintf("RemoveInstances: No sufficient instances to Kill - " +
+			"Instances to kill %d Total Instances %d", count, len(instanceIds)))
+	}
+	instanceList := make([]int32, count)
+	i := 0
+	for k := range instanceIds {
```
Comment (rdelval) [inline realis.go]: Could we do something like `instanceList = instanceIds[:count-1]` instead of this loop?

Comment (sunilmvn) [inline realis.go]: Can't do that, instanceIds is a map
cannot slice instanceIds (type map[int32]bool)

**Comment by rdelval** [comment]
Woops, good catch, thanks!

**Pull request merged**